src/_guides/libraries/library-tour.md

```diff
@@ -1757,6 +1828,7 @@ To learn more about the Dart language, see the
 [NativeFinalizer]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/NativeFinalizer-class.html
 [NoSuchMethodError]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-core/NoSuchMethodError-class.html
 [Object]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-core/Object-class.html
+[`ParallelWaitError`]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-async/ParallelWaitError-class.html
```

Comment:
why does this one use backticks when the others don't?

Reply:
I'd argue that the class names that _aren't_ back ticked need to change?